HopsanGUI/GUIConnector.cpp

```diff
@@ -1002,9 +1002,9 @@ void Connector::setUnHovered()
 
 
 //! @brief Asks the parent system to delete the connector
-void Connector::deleteMe(UndoStatusEnumT undo)
+void Connector::deleteMe(UndoStatusEnumT undo, bool force)
```

Comment:
Yes, this should be used with caution. I have not seen any problems in this case. If we don't want to reconfigure directly from the properties dialog we would need some `reconfigureLater()` function, that runs directly after the parameter is changed. This would on the other hand make the "Apply" button useless for the reconfiguration parameter.

Reply:
Hmm I guess force delete is acceptable for connectors.
But I dont know what happens if you have multiple properties boxes open for different components, and then  a connector connecting them disappears.  Likely the startvalue case is handled by the other check (haveParameter).

And what happens if you have the port location editor open. Not sure if can handle removed ports, must test that.

Reply:
A solution could be to make the connector broken instead of removing it? Port location editor could still be a problem though.

Reply:
Done. Connectors now become broken if port is removed when changing "path" parameter. It seems to work. I have tried with having the other component properties dialog open, and it did not crash. When port position dialog is open parameters cannot be changed anyway.